dimagi/pages/templates/sections/covid_19/solutions.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="solutions align-center blue-theme"
+         id="solutions">
+
+  <h3 class="normalcase font-weight-bold">
+      {% blocktrans %}
+      How is CommCare used for COVID-19 response?
+      {% endblocktrans %}
+  </h3>
+  <div class="content pad-top-medium">
+    <div class="grid-2-at-small grid-4-at-large grid-col-space-medium grid-row-space-small">
+      {% include "sections/covid_19/solutions/screening_and_triage_protocols.html" %}
+      {% include "sections/covid_19/solutions/surveillance_and_contact_tracing.html" %}
+      {% include "sections/covid_19/solutions/information_dissemination.html" %}
+      {% include "sections/covid_19/solutions/lab_tracking_and_logistics.html" %}
+    </div>{# .grid-2-at-small #}
+  </div>{# .content #}
+
+    <p class="align-center">
+      <a class="button uppercase {% click_body 'commcare' %}"
+         href="{% url 'commcare' %}">
+        {% blocktrans %}
+          Learn about CommCare
+        {% endblocktrans %}
+      </a>
+    </p>
+</section>
```

Comment:
always make sure files have a newline at the end. also this entire file needs to have the indentation adjusted according to our standards.